Configuration/Generator/python/XiMinus_13p6TeV_SoftQCDInel_TuneCUEP8M1_cfi.py

```diff
@@ -0,0 +1,38 @@
+# Found 60 output events for 2000 input events.
+# Filter efficiency = 0.03
+# Timing = 0.298047 sec/event
+# Event size = 410.3 kB/event
+
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```

Comment:
Same question as OmegaMinus.

Reply:
Also, please use `Pythia8ConcurrentGeneratorFilter` to enable multithreading.

Reply:
Actually, perhaps with EvtGen this is not possible.

Reply:
This one should be used with `Pythia8ConcurrentGeneratorFilter`